pages/Discord-Rules.md

```diff
@@ -126,33 +101,25 @@ You are NOT allowed to post a link to your discord server.
 
 This channel is **ONLY** for showcasing your Server, any other messages will be deleted.
 
-### #bug-discussions
-This channel is only for discussions around bugs, **not for reporting them.**<br>
+### #bugs
+This channel is for discussions around bugs, **not for reporting them.**<br>
 For reporting bugs, please consult [How to report bugs](https://github.com/Slimefun/Slimefun4/wiki/How-to-report-bugs).
 
 Before posting here, please follow this short guide:<br>
 1. Run `/sf versions`
 2. Capture the output of that command and include it in your message.
-3. Are there any errors? If so, then post them via https://pastebin.com (Posting an error in the chat will result in auto-deletion)
-4. See if there are newer versions of Slimefun and CS-CoreLib available. (https://thebusybiscuit.github.io/builds/)
-5. Try to be as clear as possible, the minimum is two sentences. ("X does not work" is not helpful)
-6. Double-check that you are using a "DEV - ..." build of Slimefun and CS-CoreLib, other builds are considered old or unofficial. **We will not accept bug reports from old or unofficial versions, RC versions will also be rejected since they are outdated.**
+3. Are there any errors? If so, then please post them via https://pastebin.com (Posting an error in the chat will result in auto-deletion)
+4. See if there is a newer version of Slimefun available. (https://thebusybiscuit.github.io/builds/)
+6. Try to be as clear as possible, the minimum is two sentences. ("X does not work" is not helpful)
+7. Double-check that you are using a "DEV - ..." build of Slimefun, any other builds are considered old or unofficial. **We will not accept bug reports from old or unofficial versions, RC versions will also be rejected since they are outdated.**
```

Comment:
```suggestion
4. See if there is a newer version of Slimefun available. (https://thebusybiscuit.github.io/builds/)
5. Try to be as clear as possible, the minimum is two sentences. ("X does not work" is not helpful)
6. Double-check that you are using a "DEV - ..." build of Slimefun, any other builds are considered old or unofficial. **We will not accept bug reports from old or unofficial versions, RC versions will also be rejected since they are outdated.**
```